### PR TITLE
Skip building cmssw in case of runtime image when testing a new image

### PR DESCRIPTION
**Pull request by aandvalenzuela**
It goes with the following job configuration: https://cmssdt.cern.ch/jenkins/job/cms-containers-run-cmssw-test/configure

### CODE REVIEW

**Comment by smuzaffar** [comment]
@aandvalenzuela , looks like the runtime images are really only `runtime` i.e. one can use it to run preinstalled cmssw. Can you please check what packages are required if we want to use it to install cmssw too?

**Comment by smuzaffar** [comment]
thinking about it again, if we want runtime images to also support installation then we have to add all the packages which we have in bootstrap seed (which are a lot). I would suggest to add a new test for runtime images which actually use cmssw installation from cvmfs i.e. the test will should not do any local cmssw installation but should use releases from cvmfs , create dev area and run short matrix and add on tests

**Comment by aandvalenzuela** [comment]
I have tested the script for the runtime image at https://cmssdt.cern.ch/jenkins/job/test-cmssw-images/19/ and for a regular image at https://cmssdt.cern.ch/jenkins/job/test-cmssw-images/20/ (I will delete the job once this PR is merged and they will run on the regular https://cmssdt.cern.ch/jenkins/job/cms-containers-run-cmssw-test/ job)

**Comment by smuzaffar** [comment]
tests looks good

File: bin/test-cmssw.sh [hunk 1]
```diff
@@ -32,107 +34,154 @@ if [ "${ARCHS}" = "" ] ; then
...
+        INSTALL_PACKAGES="${INSTALL_PACKAGES} $pkg"
+      fi
+    done
+    if [ "${INSTALL_PACKAGES}" != "" ] ; then
+      $WORKSPACE/inst/$SCRAM_ARCH/common/cmspkg $INST_OPTS ${INSTALL_PACKAGES}
+    fi
+     export cmssw_ver
+    (
+      export CMS_PATH=/cvmfs/cms-ib.cern.ch
+      export SITECONFIG_PATH=/cvmfs/cms-ib.cern.ch/SITECONF/local
```
Comment (smuzaffar) [inline bin/test-cmssw.sh:133]: these two lines are common for all archs/releases and containers. So you can move these above the arch loop

File: bin/test-cmssw.sh [hunk 2]
```diff
@@ -32,107 +34,154 @@ if [ "${ARCHS}" = "" ] ; then
...
+      echo "${SCRAM_ARCH}.${cmssw_ver}.TEST.${RES}" >> $WORKSPACE/res.txt
+    )
+    rm -rf $SCRAM_ARCH
+  else
+    touch cmssw.rel
+    $(source /cvmfs/cms.cern.ch/cmsset_default.sh >/dev/null 2>&1; scram -a $SCRAM_ARCH list -c CMSSW | grep -v '/cmssw-patch/' | grep ' CMSSW_' >cmssw.rel) || true
+    cmssw_ver=$(grep /cvmfs/cms.cern.ch/ cmssw.rel | tail -1 | awk '{print $2}' || true)
+    echo "Getting CMSSW area from /cvmfs: $cmssw_ver"
     export CMS_PATH=/cvmfs/cms-ib.cern.ch
     export SITECONFIG_PATH=/cvmfs/cms-ib.cern.ch/SITECONF/local
```
Comment (smuzaffar) [inline bin/test-cmssw.sh:170]: remove these once they are move above the arch loop

**Comment by aandvalenzuela** [comment]
Thanks @smuzaffar!

**Comment by aandvalenzuela** [comment]
We can merge it! :)

**Pull request merged**